raiden/api/rest.py

```diff
@@ -1621,6 +1621,7 @@ def initiate_payment_light(
 
     def settlement_light(self,
                          registry_address: typing.PaymentNetworkID,
+                         internal_msg_identifier: int,
```

Comment:
a few quesitons here—i am trying to wrap my head around this parameter:
- will this internal identifier _always_ be provided from the caller?
  - if so: how did we manage without this field before?
- doesn't this come from an API call? why is it called _internal_ if it comes from an API caller?

Reply:
Is the name of the parameter for identify a message from the LC, basically we didn't changed the name since we don't want to add that change to the SDK, the SDK receives that parameter and send it to us as it is again so we can identify the message in the database, this is new, before this we have the problem where we don't know how to retrieve or identify a message that the LC has signed.

Reply:
i've created issue [RLP-843](https://jirainfuy.atlassian.net/browse/RLP-843) to deal with this.